integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -695,11 +696,17 @@ private int chownInternal(String path, long uid, long gid) {
 
   @Override
```

Comment:
can you add comments here saying we only support truncate for special cases like xxx